package.json

```
@@ -18,7 +18,7 @@
     "prettify": "prettier --write \"**/*.{ts,js}\"",
     "typecheck": "tsc --noEmit",
     "cypress:open": "cypress open --config-file tests/cypress/cypress-config.js --e2e --browser chrome",
-    "cypress:run": "cypress run --config-file tests/cypress/cypress-config.js",
+    "cypress:run": "cypress run --config-file tests/cypress/cypress-config.js --browser chrome",
```

```suggestion
    "cypress:run": "cypress run --config-file tests/cypress/cypress-config.js",
```

Could we keep running this as before? as most of our plugins don't explicitly run tests in chrome?